spec/locale.html

```diff
@@ -203,38 +125,6 @@ <h1>Intl.Locale.prototype [ %Symbol.toStringTag% ]</h1>
       </p>
     </emu-clause>
 
-    <emu-clause id="sec-Intl.Locale.prototype.maximize">
```

Comment:
Intl.Locale is not a service constructor, so we should probably just follow ECMA-262 conventions—string-keyed properties alphabetically, then symbol-keyed properties alphabetically (i.e., Symbol.toStringTag last).

Reply:
> Intl.Locale is not a service constructor, so we should probably just follow ECMA-262 conventions—string-keyed properties alphabetically, then symbol-keyed properties alphabetically (i.e., Symbol.toStringTag last).

This one still stands; please move at least `Intl.Locale.prototype [ %Symbol.toStringTag% ]` to the end of "Properties of the Intl.Locale Prototype Object".